libraries/AP_HAL_ChibiOS/hwdef/crazyflie2/hwdef.dat

```diff
@@ -121,3 +121,7 @@ define MAG_BOARD_ORIENTATION ROTATION_YAW_180
 
 # no ADC pins
 define HAL_USE_ADC FALSE
+
+include ../include/minimize_fpv_osd.inc
```

Comment:
Does this board actually have OSD hardware?

Reply:
changed to minimize_features